src/libraries/Microsoft.Extensions.Hosting.Systemd/Directory.Build.Props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
The file name should be `Directory.Build.props` (lower-case props).

Reply:
nice catch!